systest/build.rs

```diff
@@ -42,7 +42,7 @@ fn main() {
     }
 
     let has_cms_h = if let Ok(version) = env::var("DEP_OPENSSL_LIBRESSL_VERSION") {
-        version != "261" && version != "262"
+        version.parse::<i32>().unwrap() < 250
```

Comment:
We only support 2.5.x and 2.6.x, so we don't need to look at the version at all I don't think.